### ISSUE TITLE
Enable MONGODB-AWS authentication on MongoDB query connector

### ISSUE THREAD

**Issue by loicsimon** [prisma/prisma#12151]

**Comment by janpio** [comment]
(I moved this issue here from `prisma-engines` as all feature requests are discussed here.)

**Comment by rohit-genvis** [comment]
@janpio is there any resolution on the above, we are exploring Prisma as an alternative to Mongoose, but this issue is a big hindrance.

**Comment by rohit-genvis** [comment]
@loicsimon are you able to resolve it by any chance?

**Comment by janpio** [comment]
No, this is not supported in Prisma yet. This issue tracks this as a feature request, if we implement it we will of course update the issue.

**Comment by hartimcwildfly** [comment]
:(

**Comment by ohabash** [comment]
Is this supported yet?

**Comment by v-rreddy** [comment]
Are you planning to include this solution in any upcoming release?
